src/mscorlib/src/System/Threading/ThreadPool.cs

```diff
@@ -36,6 +36,9 @@ internal static class ThreadPoolGlobals
         public static volatile bool vmTpInitialized;
         public static bool enableWorkerTracking;
 
+        public const int wsqNowActive = 2; // Require at least 2 full observations of empty wsq to transition to inactive
+        public static int wsqActive;
```

Comment:
These two fields could be private fields of ThreadPoolWorkQueue since that's the only one that uses them. Please use Pascal casing for the constant, even though the naming is inconsistent in this file.